test/Graphics/ConvexShape.test.cpp

```diff
@@ -42,4 +43,31 @@ TEST_CASE("[Graphics] sf::ConvexShape")
         convex.setPoint(0, {3, 4});
         CHECK(convex.getPoint(0) == sf::Vector2f(3, 4));
     }
+
+    SUBCASE("Reasonable geometric center")
+    {
+        sf::ConvexShape convex;
+        for (unsigned i = 2; i < 10; ++i)
```

Comment:
```suggestion
        for (unsigned int i = 2; i < 10; ++i)
```
We always use `unsigned int` rather than `unsigned` for historical style reasons.